### PR TITLE
Implement quick menu buttons from bottom toolbar

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch implements quick menu buttons. Those are from the bottom
toolbar buttons.

Refs: #936

### CODE REVIEW

File: src/org/triple/banana/quick_menu/QuickMenuActionProviderImpl.java [hunk 1]
```diff
@@ -19,7 +24,39 @@
 
     QuickMenuActionProviderImpl() {
         mActionMap = new HashMap<Integer, Action>() {
-            { put(R.id.dark_mode, () -> DarkModeController.get().toggle()); }
+            {
+                put(R.id.add_secret_tab, () -> BananaToolbarManager.get().addSecretTab());
```
Comment (romandev) [inline src/org/triple/banana/quick_menu/QuickMenuActionProviderImpl.java:28]: `put(R.id.add_secret_tab, BananaToolbarManager.get()::addSecretTab);` works for this?

File: src/org/triple/banana/quick_menu/QuickMenuControllerImpl.java [hunk 2]
```diff
@@ -24,8 +25,67 @@
     }
 
     private void loadButtons() {
+        mQuickMenuViewModel.getEditor().addButton(new ButtonInfo(R.id.add_secret_tab,
```
Comment (romandev) [inline src/org/triple/banana/quick_menu/QuickMenuControllerImpl.java:28]: Can we add StorageModel concept? Then, we can read the data from the storage model.

**Review by romandev — COMMENTED** [review]
Looks good to me but I suggested more things

File: src/org/triple/banana/quick_menu/QuickMenuActionProviderImpl.java (hunk 1 above)
Comment (jdragonbae) [reply src/org/triple/banana/quick_menu/QuickMenuActionProviderImpl.java]: It seems to work. Thanks for the review.

File: src/org/triple/banana/quick_menu/QuickMenuControllerImpl.java (hunk 2 above)
Comment (jdragonbae) [reply src/org/triple/banana/quick_menu/QuickMenuControllerImpl.java:27]: Sounds good to me. should it be added to this patch or another?

Comment (romandev) [reply src/org/triple/banana/quick_menu/QuickMenuControllerImpl.java:27]: Anything is okay. If you want to do it in separated patch, then, let's leave a FIXME comment.

File: src/org/triple/banana/quick_menu/QuickMenuFactoryImpl.java [hunk 3]
```diff
@@ -12,8 +12,8 @@
 class QuickMenuFactoryImpl implements QuickMenuFactory {
     @Override
     public @NonNull QuickMenuController create(@NonNull Context context) {
-        MockQuickMenuViewImpl mockView = new MockQuickMenuViewImpl(context);
-        QuickMenuViewImpl view = new QuickMenuViewImpl(context);
+        MockQuickMenuViewImpl view = new MockQuickMenuViewImpl(context);
+        // QuickMenuViewImpl view = new QuickMenuViewImpl(context);
```
Comment (romandev) [inline src/org/triple/banana/quick_menu/QuickMenuFactoryImpl.java:16]: Please leave a comment as well.

File: src/org/triple/banana/quick_menu/QuickMenuControllerImpl.java (hunk 2 above)
Comment (jdragonbae) [reply src/org/triple/banana/quick_menu/QuickMenuControllerImpl.java:28]: Thanks. Done.

File: src/org/triple/banana/quick_menu/QuickMenuFactoryImpl.java (hunk 3 above)
Comment (jdragonbae) [reply src/org/triple/banana/quick_menu/QuickMenuFactoryImpl.java]: Sorry, It was my mistake.

**Review by romandev — APPROVED** [review]
LGTM

**Pull request merged**